### ISSUE TITLE
Missed dot in extension list in _file2dict function

### ISSUE THREAD

**Issue by Danil328** [open-mmlab/mmcv#468]
https://github.com/open-mmlab/mmcv/blob/045629c2804fbf311d246fb4edbafca34129b981/mmcv/utils/config.py#L122

```
if fileExtname not in ['.py', '.json', '.yaml', 'yml']:
>           raise IOError('Only py/yml/yaml/json type are supported now!')
E           OSError: Only py/yml/yaml/json type are supported now!
```

Correct 
```
['.py', '.json', '.yaml', '.yml']
```

**Comment by hellock** [comment]
Thanks for the reporting! Could you create a PR to fix it?

**Comment by Danil328** [comment]
https://github.com/open-mmlab/mmcv/pull/469

**Comment by Danil328** [comment]
merged #469
